Healthy/Classes/ReusableViews/FormTextField/FormTextField.swift

```diff
@@ -0,0 +1,78 @@
+
+import UIKit
+
+class FormTextField: UIView {
+    
+    // MARK: - Outlets
+    //
+    
+    @IBOutlet weak private var contentView: UIView!
+    @IBOutlet weak private var titleLabel: UILabel!
+    @IBOutlet weak private var inputTextField: UITextField!
+    @IBOutlet weak private var errorLabel: UILabel!
```

Comment:
```suggestion
    @IBOutlet private weak var contentView: UIView!
    @IBOutlet private weak var titleLabel: UILabel!
    @IBOutlet private weak var inputTextField: UITextField!
    @IBOutlet private weak var errorLabel: UILabel!
```

Access modifier comes before the reference type.

Reply:
Please make sure to install swift lint until we integrate the CI